src/scenes/home/team/team.js

```diff
@@ -18,7 +19,7 @@ class Team extends Component {
 
   componentDidMount() {
     axios
-      .get('https://api.operationcode.org/api/v1/team_members.json')
```

Comment:
This will be a good change - same for the /code-schools page. I've always wondered why the production API URL was hardcoded in here.